ampd/src/handlers/solana_verify_msg.rs

```diff
@@ -63,8 +66,9 @@ struct PollStartedEvent {
     poll_id: PollId,
     source_gateway_address: String,
     messages: Vec<Message>,
-    // participants: Vec<TMAddress>,
-    participants: Vec<String>,
+    participants: Vec<TMAddress>,
+    expires_at: u64
+    //participants: Vec<String>,
```

Comment:
Was this changed due to manual tests ?

Reply:
Yeah. I'll delete all commented code and debug messages, before merging to the `eigerco` branch.